lint_requirements.txt

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
you are missing isort (feel free to run it on the whole codebase if it does not pass right now, but if you do make one commit with only those changes)